### ISSUE TITLE
update node-fetch to v3.x

### ISSUE THREAD

**Issue by jimmywarting** [matthew-andrews/isomorphic-fetch#200]

**Comment by cosmolightfoot** [comment]
Any progress on this one? 

Our company's security scans are showing vulnerabilities in node-fetch and are blocking deploys based on this package based on the sub-dependency.
node-fetch v3.1.1 solves the issue.

**Comment by borisovg** [comment]
Apparently `node-fetch` v3 breaks non-ESM applications: https://github.com/node-fetch/node-fetch/issues/1263

**Comment by bhavikagrawal** [comment]
I am also stuck for a week now and have not found any solution yet.

**Comment by jimmywarting** [comment]
@bhavikagrawal what are you stuck with?
anything i can do to help?

**Comment by RishikeshDarandale** [comment]
@jimmywarting , I am also stuck in one of the [issue](https://github.com/auth0/auth0-react/issues/323), the fix for this one is get updated to `node-fetch` - `v3.x`.

**Comment by shazron** [comment]
[2.6.7](https://github.com/node-fetch/node-fetch/releases/tag/v2.6.7) also has the security patch

**Comment by RishikeshDarandale** [comment]
@shazron , Yes it's patched in `2.6.7`, but there is an issue using `2.x` in tandem with `auth-react` package. Please see [here](https://github.com/auth0/auth0-react/issues/323), thus upgrade `3.x` would be the **best** way for me. Yes, there is a workaround using with `2.x`, but going with `3.x` is better way for me.
